<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SVCoSimCO2VentilationDialog</class>
 <widget class="QDialog" name="SVCoSimCO2VentilationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Co-Simulation with CO2 Balance and Ventilation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelCheckSummary">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>The button below creates an FMU with the CO2 balances for each zone and the ventilation control algorithms according to the settings made in the zone templates. It also creates the matching NANDRAD FMU and the co-simulation project file for MASTERSIM.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonGenerate">
     <property name="text">
      <string>Generate FMU and Co-Simulation project</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonLaunchMasterSim">
     <property name="text">
      <string>Open Co-Simulation project in MasterSim...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
